packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporter.ts

```diff
@@ -38,6 +40,10 @@ export class CollectorExporter extends CollectorExporterBase<
     window.removeEventListener('unload', this.shutdown);
   }
 
+  defaultURL(url: string | undefined) {
```

Comment:
I would change the name to describe some action as `defaultURL` sounds like property and not a method. Maybe `setDefaultUrl` ?

Reply:
For me, `getDefaultUrl` makes more sense than `setDefaultUrl`. We are not setting anything as a part of this method. WDYT @obecny @davidwitten ?

Reply:
can be getDefaultUrl, chooseDefaultUrl etc. just something that sounds like action / method :)

Reply:
@obecny @mayurkale22 Just fixed that, thanks!